source/stylesheets/refills/unstyled/_footer.scss

```diff
@@ -0,0 +1,64 @@
+// Uncomment this for a sticky footer.
+//
+// .wrapper-for-content-outside-of-footer {
+//   min-height: 100%;
+//   height: auto !important;
+//   height: 100%;
+//   margin: 0 auto -4em;
+// }
+
+// html, body {
+//  height: 100%;
+// }
+
+// .wrapper-for-content-outside-of-footer:after {
+//   content: "";
+//   display: block;
+// }
+
+// .footer, .wrapper-for-content-outside-of-footer:after {
+//   $large-screen: em(860) !default;
+//   height: 12em;
+
+//   @include media($large-screen) {
+//     height: 4em;
+//   }
+// }
+
+.footer {
+  $large-screen: em(860) !default;
+
+  .footer-logo {
+    margin-right: 1em;
+    margin-bottom: 1em;
+
+    @include media($large-screen) {
+      float: left;
+      margin-bottom: 0;
+    }
+  }
+
+  .footer-logo img {
```

Comment:
Merge rule `.footer-logo img` with rule on line 31<br>Selector should have depth of applicability no greater than 2, but was 3